### PR TITLE
[15.0][MIG] mass_mailing_custom_unsubscribe_event: Migration to version 15.0

### PR DESCRIPTION
**Pull request by ernesto-garcia-tecnativa**
@tecnativa TT41876

### CODE REVIEW

**Comment by ernesto-garcia-tecnativa** [comment]
@pedrobaeza @victoralmau please review

**Review by victoralmau — COMMENTED** [review]
Can you check tests?

**Comment by ernesto-garcia-tecnativa** [comment]
> Can you check tests?

Sorry, I tagged you before end the test here, because in my local works fine, but I working in resolve this

**Comment by ernesto-garcia-tecnativa** [comment]
@chienandalu @pedrobaeza please review

**Review by chienandalu — APPROVED** [review]
Thanks! :+1:

File: mass_mailing_custom_unsubscribe_event/tests/test_ui.py [hunk 1]
```diff
@@ -68,12 +70,11 @@ def setUp(self):
...
                 "mailing_model_id": self.env.ref("event.model_event_registration").id,
                 "mailing_domain": "[['id','=',%s]]" % self.event_registration_1.id,
-                "reply_to_mode": "email",
+                "reply_to_mode": "update",
                 "subject": "Test 1",
             }
         )
-        self.mailing_1._onchange_model_and_list()
-        # HACK https://github.com/odoo/odoo/pull/14429
+
```
Comment (pedrobaeza) [inline mass_mailing_custom_unsubscribe_event/tests/test_ui.py]: Don't add empty lines inside a method.

**Review by pedrobaeza — COMMENTED** [review]
The pre-commit commit has an abnormal commit message (how many commit words, hehe).

**Comment by ernesto-garcia-tecnativa** [comment]
> The pre-commit commit has an abnormal commit message (how many commit words, hehe).

I removed commit, sorry

**Review by pedrobaeza — APPROVED** [review]
/ocabot merge nobump

**Pull request merged**